### ISSUE TITLE
Master's `intro_scicomp_python.ipynb` has broken data

### ISSUE THREAD

**Issue by ljwolf** [pysal/notebooks#20]
In [this section](http://nbviewer.ipython.org/github/pysal/notebooks/blob/master/notebooks/intro_scicomp_python.ipynb#Hands-on-example), we read data from [`http://ubuntuone.com/4oIpVJDCpdREhzNdSRwMx8`](http://ubuntuone.com/4oIpVJDCpdREhzNdSRwMx8), which no longer exists. 

I'm not sure what the dataset was supposed to be, but it looks like it was a set of tweets?

**Comment by marwahaha** [comment]
According to @darribas it seems that there is a set of tweets but it is privately licensed.. 

(via https://github.com/pysal/pysal/issues/151)

**Comment by marwahaha** [comment]
My opinion is that this renders the entire https://github.com/pysal/notebooks/blob/master/notebooks/intro_scicomp_python.ipynb notebook and 
the last part of the https://github.com/pysal/notebooks/blob/master/notebooks/PySAL_io.ipynb notebook unusable ---  my immediate reaction to a Notebook is to reproduce all the output (i.e. run all the cells). Maybe we can find another source of tweets?

**Comment by darribas** [comment]
Yes! I agree not having the data available is suboptimal. I'm not aware of any directly available Twitter dataset we could use for this but, if you find any that's linkable, it'd be great to adapt it I think. Of course, PR's most welcome! ;-)
